### ISSUE TITLE
SOLR image - schema.xml is using the incorrect values

### ISSUE THREAD

**Issue by g7morris** [Islandora-Collaboration-Group/ISLE#185]
#### Issue description

The current SOLR image v.1.1 is using incorrect values in its `schema.xml` file. 

It should be using `1.5` instead of `1.1` (default stock value) this allows multi-field indexing amongst other abilities. Issues could arise for users using this lower default value. 

Recommend updating Dockerfile to change this value. 

It appears this was missed potentially when upgrading the ISLE Solr image from ISLE v1.0 to v 1.1

#### Steps to reproduce the issue

1.  Review here: https://github.com/Islandora-Collaboration-Group/isle-solr/blob/6ea451d49d53637f4a800f82959d4a9f79684418/Dockerfile **Line 15** which indicates the clone of DG's `4.10.x` branch of their `basic-solr-config` repo 

2. Review DG's `4.10.x` branch of their `basic-solr-config` repo https://github.com/discoverygarden/basic-solr-config/blob/4.10.x/conf/schema.xml (Note the version #)

3. Compare to https://github.com/Islandora-Collaboration-Group/ISLE/blob/ISLE-v1.0/solr/solr/schema.xml **Line 32**

#### What's the expected result?

- That the `schema.xml` file would have the following set:

```
<schema name="DemoOnSolr" **version="1.5">**
  <!-- attribute "name" is the name of this schema and is only used for display purposes.
       Applications should change this to reflect the nature of the search collection.
       version="1.1" is Solr's version number for the schema syntax and semantics.  It should
       not normally be changed by applications.
       1.0: multiValued attribute did not exist, all fields are multiValued by nature
       1.1: multiValued attribute introduced, false by default -->
```

#### What's the actual result?

- The `schema.xml` file uses the default stock supplied from cloning the Discovery Garden `basic-solr-config` git repo (branch `4.10.x`)

```
<schema name="DemoOnSolr" **version="1.1">**
  <!-- attribute "name" is the name of this schema and is only used for display purposes.
       Applications should change this to reflect the nature of the search collection.
       version="1.1" is Solr's version number for the schema syntax and semantics.  It should
       not normally be changed by applications.
       1.0: multiValued attribute did not exist, all fields are multiValued by nature
       1.1: multiValued attribute introduced, false by default -->
```

**Comment by g7morris** [comment]
@br2490 `schema.xml` looks good! Ship it! (_thanks_)

**Comment by br2490** [comment]
This is folded into and discussed in #186

**Comment by g7morris** [comment]
@br2490 Getting odd behavior on this one. 

I shut down my running containers. Did a `docker-compose pull`, got the new SOLR image BUT when I ...

* `docker exec -it isle-solr-container bash`
* `cat /usr/local/solr/collection1/conf/schema.xml`

I'm still seeing the 1.1 not 1.5 here

```
<schema name="DemoOnSolr" version="1.1">
  <!-- attribute "name" is the name of this schema and is only used for display purposes.
       Applications should change this to reflect the nature of the search collection.
       version="1.1" is Solr's version number for the schema syntax and semantics.  It should
       not normally be changed by applications.
       1.0: multiValued attribute did not exist, all fields are multiValued by nature
       1.1: multiValued attribute introduced, false by default -->```

**Comment by g7morris** [comment]
@br2490 I think I know what the issue is.  In the standard default, ISLE setup and `docker-compose.yml` file there is this entry on line 80 / 81

```
volumes:
      - isle-solr-data:/usr/local/solr
```

The file (`schema.xml`) won't change as it is persisted into a volume. 

I think I'd recommend changing the docker-compose.yml to something like 

```
volumes:
      - isle-solr-data:/usr/local/solr/collection1/data
```

This way ISLE can push changes to schemas, configs etc without forcing users to destroy their data.

If folks need to change their schemas, solrconfig.xml etc then a bind mount will help them.

**Comment by g7morris** [comment]
@br2490 Confirming that once I delete the associated solr volume `isle-solr-data:/usr/local/solr` which contains the index, perform a `docker-compose pull` and then `docker-compose up -d` I see the correct `schema.xml`. Though it means I have to reindex.

**Comment by g7morris** [comment]
@br2490 I'm leaving the ticket open as a question re the architectural changes but I don't feel this is a blocker any more given the changes.

**Comment by g7morris** [comment]
This issue have been fixed as of ISLE 1.2+ Closing ticket.
